framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -850,7 +945,11 @@ private boolean skipNode(NodeWithAccessModifiers<?> node) {
           processField((FieldDeclaration) decl, (VariableElement) elt);
           break;
         case ENUM_CONSTANT:
-          processEnumConstant((EnumConstantDeclaration) decl, (VariableElement) elt);
+          if (decl instanceof FieldDeclaration) {
+            processField((FieldDeclaration) decl, (VariableElement) elt);
+          } else if (decl instanceof EnumConstantDeclaration) {
+            processEnumConstant((EnumConstantDeclaration) decl, (VariableElement) elt);
+          }
```

Comment:
I think it would be best to add a default/fallthrough `else` that throws an exception, rather than silently doing nothing.